tic-tac-toe.js

```diff
@@ -1,7 +1,76 @@
 function TicTacToe() {
-
+  this.winConditions = [[0,1,2], [3,4,5], [6,7,8], [0,3,6], [1,4,7], [2,5,8],[0,4,8], [6,4,2]];
+  this.clickSpace();
 }
 
 TicTacToe.prototype = {
-  
-}
+  // check which player and return the appropriate mark
+  drawMark: function(player){
+    return player === 1 ? '🍍' : '🌺';
```

Comment:
I think it would be nice if the `player` variable was stored on the TicTacToe game itself as opposed to being passed around as a parameter. This might help simplify some logic throughout.